plugins/modules/aci_l3out_hsrp_interface_profile.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_hsrp_interface_profile
```

Comment:
Isn't this all module representing an class object with a 1:1 relationship with `l3extLIfP`? Shouldn't we add this in child_classes of `aci_l3out_logical_interface_profile` module?

Reply:
This is a special case where hsrp_interface_profile is a placeholder object. It cannot be identified by a name and hence can't be used as a child of l3out ansible module. The presence of this module in the playbook checks the HSRP box in the UI.